project/AppDependencies.scala

```diff
@@ -1,20 +1,20 @@
-import play.core.PlayVersion
-import play.sbt.PlayImport._
-import sbt.Keys.libraryDependencies
+import play.core.PlayVersion.current
 import sbt._
 
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "4.2.0"
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "4.1.0"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-27"   % "4.2.0"  % Test,
-    
-    "org.scalatest"           %% "scalatest"                % "3.2.5"  % Test,
-    "com.typesafe.play"       %% "play-test"                % PlayVersion.current  % Test,
-    "com.vladsch.flexmark"    %  "flexmark-all"             % "0.36.8" % "test, it",
-    "org.scalatestplus.play"  %% "scalatestplus-play"       % "4.0.3"  % "test, it"
+    "uk.gov.hmrc"             %% "bootstrap-test-play-27"   % "4.1.0"                 % Test,
```

Comment:
dependency downgrade

Reply:
And some other ones in this file

Reply:
The reason for these is that we are copying files from another service which is using these versions. I wanted to make sure they were aligned which ensures no compatibility issues. If needed in the future we can come along and update all the backends together

Reply:
Ok, good enough for me